File: CS/WinWebSolution.Module/ShowAssignedDocumentsDetailViewController.cs
using System;
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.ExpressApp.SystemModule;

namespace WinWebSolution.Module {
    public partial class ShowAssignedDocumentsDetailViewController : ViewController {
        public ShowAssignedDocumentsDetailViewController() {
            InitializeComponent();
            RegisterActions(components);
        }
        private void popupWindowShowAction1_CustomizePopupWindowParams(object sender, CustomizePopupWindowParamsEventArgs e) {
            ContactBase obj = View.CurrentObject as ContactBase;
            if (obj == null) {
                throw new ArgumentNullException("View.CurrentObject");
            }
            ListView lv = Application.CreateListView(View.ObjectSpace, typeof(DocumentBase), false);
            lv.CollectionSource.Criteria["AssignedDocuments"] = obj.AssignedDocumentsCriteria;
            e.View = lv;
            e.DialogController.SaveOnAccept = false;
            e.DialogController.FrameAssigned +=new EventHandler(DialogController_FrameAssigned);

        }
        private void DialogController_FrameAssigned(object sender, EventArgs e) {
            DialogController dc = (DialogController)sender;
            dc.AcceptAction.Active["AssignedDocuments"] = false;
            dc.Frame.GetController<NewObjectViewController>().NewObjectAction.Active["AssignedDocuments"] = false;
        }
    }
}

File: VB/WinWebSolution.Module/ShowAssignedDocumentsDetailViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.ExpressApp.SystemModule

Namespace WinWebSolution.Module
	Partial Public Class ShowAssignedDocumentsDetailViewController
		Inherits ViewController
		Public Sub New()
			InitializeComponent()
			RegisterActions(components)
		End Sub
		Private Sub popupWindowShowAction1_CustomizePopupWindowParams(ByVal sender As Object, ByVal e As CustomizePopupWindowParamsEventArgs) Handles popupWindowShowAction1.CustomizePopupWindowParams
			Dim obj As ContactBase = TryCast(View.CurrentObject, ContactBase)
			If obj Is Nothing Then
				Throw New ArgumentNullException("View.CurrentObject")
			End If
			Dim lv As ListView = Application.CreateListView(View.ObjectSpace, GetType(DocumentBase), False)
			lv.CollectionSource.Criteria("AssignedDocuments") = obj.AssignedDocumentsCriteria
			e.View = lv
			e.DialogController.SaveOnAccept = False
			AddHandler e.DialogController.FrameAssigned, AddressOf DialogController_FrameAssigned

		End Sub
		Private Sub DialogController_FrameAssigned(ByVal sender As Object, ByVal e As EventArgs)
			Dim dc As DialogController = CType(sender, DialogController)
			dc.AcceptAction.Active("AssignedDocuments") = False
			dc.Frame.GetController(Of NewObjectViewController)().NewObjectAction.Active("AssignedDocuments") = False
		End Sub
	End Class
End Namespace